src/renderer/views/pool/PoolDetailsView.tsx

```diff
@@ -44,12 +48,27 @@ const renderInitialView = () => <PoolDetails {...defaultDetailsProps} />
 export const PoolDetailsView: React.FC = () => {
   const {
     service: {
-      pools: { selectedPoolDetail$, priceRatio$, selectedPricePoolAssetSymbol$ }
+      pools: { selectedPoolDetail$, priceRatio$, selectedPricePoolAssetSymbol$ },
+      setSelectedPoolAsset
     }
   } = useMidgardContext()
 
   const intl = useIntl()
 
+  const { asset } = useParams<PoolDetailRouteParams>()
+
+  const oRouteAsset = useMemo(() => O.fromNullable(assetFromString(asset.toUpperCase())), [asset])
+
```

Comment:
We do need an error handling in case of `oRouteAsset` is `O.none` (invalid asset string in route).

You can do it as follow:
- Create a `const routeAssetRD = RD.fromOptoin(oRouteAsset, () => `Invalid pool asset in route: ${asset}` /* we might have already an i18n for it */ )
- Use `RD.combine(routeAssetRD, poolDetailRD)` in line 79 to handle error cases for both: `routeAssetRD` + `poolDetailRD`